src/internal/pfsdb/commits.go

```diff
@@ -845,21 +939,30 @@ func listCommitPage(ctx context.Context, tx *pachsql.Tx, limit, offset int, filt
 		}
 		switch key {
 		case CommitBranches:
-			conditions = append(conditions, fmt.Sprintf("commit.%s IN (SELECT id FROM pfs.branches WHERE name IN (%s))", string(key), strings.Join(quotedVals, ",")))
+			conditions = append(conditions, fmt.Sprintf("(commit.%s IN (SELECT id FROM pfs.branches WHERE name IN (%s)))", string(key), strings.Join(quotedVals, ",")))
```

Comment:
I think it would be much safer to take these values as `[]any` append them to a slice of arguments at the same time as you are building the query.  Then pass your accumulated `args...` to SelectContext, and let the driver do the escaping.

Reply:
I think you can still inject through the quotedVals above.

Reply:
Gotcha, will do!